react/components/QuantityStepper.css

```diff
@@ -1,3 +1,8 @@
 .inputContainer {
   width: 6rem;
 }
+
+.quantityInput{}
+.quantityButton {}
+.quantityButtonDecrease {}
+.quantityButtonIncrease {}
```

Comment:
Please create the handles using the `useCssHandles` hook 🙏 
```suggestion
```

Reply:
Add these handles on documentation